spec/schema.json

```diff
@@ -525,6 +540,225 @@
         "start_date"
       ]
     },
+    "TrustData": {
+      "type": "object",
+      "properties": {
+        "trust_id": {
+          "type": "string"
+        },
+        "trust_name": {
+          "type": "string"
+        },
+        "trust_creation_date_day": {
+          "type": "string"
+        },
+        "trust_creation_date_month": {
+          "type": "string"
+        },
+        "trust_creation_date_year": {
+          "type": "string"
+        },
+        "unable_to_obtain_all_trust_info": {
+          "type": "boolean"
+        },
+        "INDIVIDUAL": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IndividualBeneficialOwner"
+          }
+        },
+        "CORPORATE": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CorporateBeneficialOwner"
+          }
+        },
+        "HISTORICAL_BO": {
+          "type": "string"
+        }
+      }
+    },
+    "IndividualBeneficialOwner": {
```

Comment:
the name IndividualBeneficialOwner is too similar to the existing BeneficialOwnerIndividual type. I can see in the package structure that the classes are separated into a 'trust' package, but in the schema I think there is no way of knowing what the difference is.  I would recommend prefixing the IndividualBeneficialOwner and CorporateBeneficialOwner with 'Trust' to make it clear what those classes are for - TrustIndividualBeneficialOwner and TrustCorporateBeneficialOwner.

Reply:
The swagger spec represents the JSON that would be generated via the spreadsheet, and this is one of the fields coming from the spreadsheet